ui/src/components/Canvas.tsx

```diff
@@ -970,10 +970,10 @@ export function Canvas() {
           if (
             mousePos.x < currentParent.positionAbsolute!.x ||
             mousePos.x >
-              currentParent.positionAbsolute!.x + currentParent.width! ||
+            currentParent.positionAbsolute!.x + currentParent.width! ||
             mousePos.y < currentParent.positionAbsolute!.y ||
             mousePos.y >
-              currentParent.positionAbsolute!.y + currentParent.height!
+            currentParent.positionAbsolute!.y + currentParent.height!
```

Comment:
We seem to be still on a different formatter (we used [prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode)). Can you double check?

Reply:
A configuration file may be needed in ui's fold.

Reply:
Thanks for the suggestions! Since we have nothing to customize prettier, we just left it empty (which should apply the default options). Do you think this would work or not? Or did you use prettier but got this different formatting?